src/module/party/party-xp.js

```diff
@@ -73,7 +82,8 @@ export class OsePartyXP extends FormApplication {
     });
   }
 
-  _dealXP(ev) {
+  // eslint-disable-next-line no-underscore-dangle
+  _dealXP() {
```

Comment:
we can use # here instead, no?

Reply:
We should be able to use `#` to mark private class members anywhere that Foundry doesn't expect a dangling underscore. I'd like to do this in a separate PR, since there are so many instances of things like this to change.